protokt-codegen/src/main/kotlin/com/toasttab/protokt/Main.kt

```diff
@@ -73,6 +80,15 @@ private fun generate(
                     protoFileList,
                     respectJavaPackage(params)
                 ),
+                protoFileList.associate {
+                    it.name to
+                        resolvePackage(
+                            it,
+                            it.name !in filesToGenerate ||
```

Comment:
If we're ignoring the Java package option, we still need to reference any descriptor dependencies _not_ generated in this run that _did_ respect the Java package.